wom/src/main/scala/wom/graph/GraphNodePort.scala

```diff
@@ -50,6 +51,12 @@ object GraphNodePort {
     }
   }
   case class GraphNodeOutputPort(override val identifier: WomIdentifier, womType: WomType, graphNode: GraphNode) extends OutputPort
+  object ExpressionBasedOutputPort {
+    def apply(localName: LocalName, womType: WomType, graphNode: GraphNode, expression: WomExpression): ExpressionBasedOutputPort = {
+      ExpressionBasedOutputPort(WomIdentifier(localName, graphNode.identifier.fullyQualifiedName.combine(localName.value)), womType, graphNode, expression)
+    }
+  }
+  case class ExpressionBasedOutputPort(override val identifier: WomIdentifier, womType: WomType, graphNode: GraphNode, expression: WomExpression) extends OutputPort
```

Comment:
OOI, what's happening here?

Doesn't seem super-related to the aborts stuff?

Reply:
Yeah this is what we discussed about having a port that contains the expression for task outputs so that we can return a `Map[OutputPort, WomValue]` when evaluating outputs and avoid having Cromwell do some string matching to find the right one. It has nothing to do with abort but I sneaked it in here.